lib/wordcount-view.coffee

```diff
@@ -15,7 +15,10 @@ class WordcountView extends View
   update_count: (editor) ->
     text = @getCurrentText editor
     [wordCount, charCount] = @count text
-    @divWords.innerHTML = "#{wordCount || 0} W | #{charCount || 0} C"
+    if atom.config.get('wordcount.hidechars')
+      @divWords.innerHTML = "#{wordCount || 0} W"
+    else
+      @divWords.innerHTML = "#{wordCount || 0} W | #{charCount || 0} C"
```

Comment:
I think it's ok to handle it this way. Another possible solution would be to split the code that word and char count are appended separately, and then you could do it in a one line change. like:

```coffee
xyz.append(" | #{charCount || 0} C") unless atom.config.get('wordcount.hidechars')
```